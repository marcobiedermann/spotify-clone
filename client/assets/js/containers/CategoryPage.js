import { connect } from 'react-redux';
import { bindActionCreators } from 'redux';
import { fetchCategory } from '../actions/browse';
import CategoryPage from '../pages/Browse/Categories/Category';

const mapStateToProps = state => ({
  ...state,
  category: state.browse.category,
});

const mapDispatchToProps = dispatch => bindActionCreators(
  {
    fetchCategory,
  },
  dispatch,
);

export default connect(
  mapStateToProps,
  mapDispatchToProps,
)(CategoryPage);
