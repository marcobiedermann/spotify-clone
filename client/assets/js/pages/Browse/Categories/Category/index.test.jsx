import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import CategoryPage from '.';

it('renders correctly', () => {
  const tree = renderer
    .create(<MemoryRouter><CategoryPage /></MemoryRouter>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
