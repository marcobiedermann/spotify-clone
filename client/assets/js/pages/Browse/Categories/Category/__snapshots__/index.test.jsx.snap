// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <figure>
    <figcaption>
      <a
        href="/browse/categories//playlists"
        onClick={[Function]}
      >
        
      </a>
    </figcaption>
  </figure>
</div>
`;
